schemas/activity/tissueCulturePreparation.schema.tpl.json

```diff
@@ -2,26 +2,24 @@
   "_type": "https://openminds.ebrains.eu/specimenPrep/TissueCulturePreparation",
   "_extends": "/core/schemas/research/ExperimentalActivity.schema.tpl.json",
   "required": [
-    "input",
-    "output",
-    "cultureType",
-    "cultureMedium"
+    "cultureMedium",
+    "cultureType"
   ],
   "properties": {
     "cultureMedium": {
-      "_instruction": "Solution the culture was maintained in prior to the experiment",
+      "_instruction": "Add the culture medium used during this tissue culture preparation.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/chemicals/ChemicalMixture"
       ]
     },
     "cultureType": {
-      "_instruction": "Type of culture (e.g. primary, secondary)",
+      "_instruction": "Add the cell culture type of the resulting tissue cell culture.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/controlledTerms/CellCultureType"
       ]
     },
     "input": {
-      "_instruction": "Add the state of the subject(s) or tissue(s) from which the culture was prepared.",
+      "_instruction": "Add the state of the specimen that was used to prepare the culture from during this activity.",
```

Comment:
this reads weird... suggestion: "Add the state of the specimen that was used to prepare the culture during this activity."

Reply:
OR "Add the state of the specimen that was used to prepare the culture from this activity." ??